mongodb/mongodb_handler.py

```diff
@@ -0,0 +1,199 @@
+from typing import Union, Tuple
+from mongoengine import DoesNotExist, MultipleObjectsReturned, NotUniqueError, ValidationError, OperationError
+from mongodb.mongodb_models.bills_model import Bill
+from mongodb.mongodb_models.patron_model import PatronDocument
+from mongodb.mongodb_models.library_item_model import LibraryItemDocument
+
+
+def update_item_status(isbn, is_borrowed) -> Tuple[bool, str]:
+    """
+    Update the status of a library item identified by its ISBN.
+
+    Args:
+        isbn (str): The ISBN of the library item.
+        is_borrowed (bool): True if the item is borrowed, False if it is returned.
+
+    Returns:
+        Tuple[bool, str]: A tuple indicating success (True) or failure (False), and an error message if applicable.
+    """
+    try:
+        library_item = LibraryItemDocument.objects.get(isbn=isbn)
+        library_item.is_borrowed = is_borrowed
+        library_item.save()
+        return True, ""
+    except DoesNotExist:
+        return False, f"Library item with ISBN {isbn} does not exist."
+    except MultipleObjectsReturned:
+        return False, f"Multiple items found with ISBN {isbn}."
+    except ValidationError as e:
```

Comment:
if you dont have a good message to write the exception just treat validation error and operation error and return the e